plugin/pkg/admission/limitranger/admission_test.go

```diff
@@ -30,14 +30,17 @@ import (
 	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/fake"
 )
 
-func getResourceList(cpu, memory string) api.ResourceList {
+func getResourceList(cpu, memory, storage string) api.ResourceList {
```

Comment:
we follow the same pattern in validation_test.go, i just think its hard to read 3 strings ;-)